shared/actions/push/index.js

```diff
@@ -50,6 +50,14 @@ function* pushNotificationSaga(notification: Constants.PushNotification): SagaGe
   if (payload && payload.userInteraction) {
     if (payload.type === 'chat.newmessageSilent') {
       console.info('Push notification: silent notification received')
+      if (payload.x && payload.x > 0) {
```

Comment:
Could we put this after the call into Go? I just want to make sure if that process is slow we also don't show the notification.